ui/src/state/storage/upload.ts

```diff
@@ -92,6 +105,23 @@ export const useFileStore = create<FileStore>((set, get) => ({
     const { key, file } = upload;
     updateStatus(uploader, key, 'loading');
 
+    const compressionOptions = {
+      maxSizeMB: 1,
+      maxWidthOrHeight: 1920,
```

Comment:
will this mess with the sizes we record from the input?

Reply:
Good point. I think it would if the image is larger than 1920x1920. Looking into it more now.

Reply:
Looks like setting this value is unnecessary, I'm going to remove it for now.